docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,170 @@
+
+```js
+before(() => {
+  Cypress.Screenshot.defaults({
+    capture: 'viewport', // or fullPage
+  })
+})
+```
+
+Add a control for a specific screenshot to only capture the application under test.
+
+```js
+it('sign in page', () => {
+  cy.visit('/sign-in')
+  cy.screenshot({
+    capture: 'viewport', // or fullPage
+  })
+})
+```
+
+### Video controls
+
+#### Disable video upload
+
+You can delete videos that are recorded when you are passing `video: true` by following [this guide to control what is uploaded to Cypress Cloud](/guides/guides/screenshots-and-videos#Control-which-videos-to-keep-and-upload-to-Cypress-Cloud).
+
+### Cypress Command Log controls
+
+#### Turn off Cypress Command Log rendering
+
+You can disable showing the entirety of the Cypress Command Log as well as the Application under Test (AUT) controls in your tests to prevent all test content from being shown and recorded in any screenshot or video by passing `--no-runner-ui` via the CLI while running Cypress tests [See Cypress Command Line options.](/guides/guides/command-line#Options). This means that the AUT will be the only thing that is rendered while running tests, so the only thing that is captured in screenshots and videos.
+
+Note that when `--no-runner-ui` is set, data from the Cypress Command Log will still be sent to Cypress Cloud to be generated and displayed in Test Replay if it is enabled. See [Disable Test Replay](/guides/cloud/debugging/test-replay#Opt-out-of-Test-Replay) for more information.
+
+#### Mask data in the Cypress Command Log
+
+There are multiple ways to mask data in the Cypress Command Log. Note that when these strategies are used, data within test definitions will still be sent to Cypress Cloud.
+
+##### `log: false` for Cypress commands
+
+You can pass `log: false` as an option to Cypress-specific commands in your tests to prevent the data from being shown visibly in the Cypress Command Log and being shown in any captured screenshots or videos.
+
+##### Disable logging of XHR/Fetch requests
+
+By default, Cypress logs all requests that match any [`cy.intercept()`](/api/commands/intercept) in the Cypress Command Log, as well as all XMLHttpRequests and fetch requests. You can use `cy.intercept()` to disable these logs by passing `{ log: false }` in the second parameter. [See `cy.intercept()` docs.](/api/commands/intercept#Disabling-logs-for-a-request)
+
+```js
+before(() => {
+  // disable Cypress's default behavior of logging all XMLHttpRequests and fetches
+  cy.intercept({ resourceType: /xhr|fetch/ }, { log: false })
+})
+```
+
+##### Mask Cypress commands in the Cypress Command Log
+
+Users can mask passwords in the Cypress Command Log by using a custom command like the code below. [See our guide on masking the contents of the `.type()` command.](/api/cypress-api/custom-commands#Overwrite-type-command)
+
+<DocsImage src="/img/api/custom-commands/custom-command-type-masked-password.png" />
```

Comment:
This screenshot is out of date. It should not show `blacklisted-host`, since it is now `blocked-host`

![image](https://github.com/cypress-io/cypress-documentation/assets/66998419/7a74a6b4-b4bd-4a58-9ad9-b0276f9f0cf2)

Reply:
hmm, yah took this from the blockHosts page.

Reply:
Updated in [`7f241b3` (#5425)](https://github.com/cypress-io/cypress-documentation/pull/5425/commits/7f241b3381fa9a5f4c0af3241453a6fc5ef27d72)